### ISSUE TITLE
unable to exec /etc/s6-overlay/scripts/startup: Permission denied

### ISSUE THREAD

**Issue by benjamin-feron** [alexjustesen/speedtest-tracker#44]
Hello,

I encounter a problem when I launch the image with an error message :

s6-rc-oneshot-run: fatal: unable to exec /etc/s6-overlay/scripts/startup: Permission denied

I launched it with docker-compose with this simple configuration :

```
services:
  speedtest-tracker:
    build: ./speedtest-tracker
```

The repo is freshly cloned in speedtest-tracker directory.

Architecture : aarch64

Can you help me ?

Thanks

**Comment by alexjustesen** [comment]
Hey @benjamin-feron I'd avoid the `main` branch atm and use a release. I'm aware of a permission issue when creating the SQLite DB during startup if you don't specify an external db.

**Comment by benjamin-feron** [comment]
Hey,

Thank you for your response

I've done a `git checkout v0.1.0-alpha.6` in the repo and the same issue persits.

```
networks:
  databases:
    external: true

services:
  speedtest-tracker:
    build: ./speedtest-tracker
    restart: unless-stopped
    networks:
      - databases
    environment:
      DB_CONNECTION: mysql
      DB_HOST: mariadb
      DB_PORT: 3306
      DB_DATABASE: speedtest-tracker_test
      DB_USERNAME: speedtest-tracker_test
      DB_PASSWORD: speedtest-tracker_test
```

`s6-rc-oneshot-run: fatal: unable to exec /etc/s6-overlay/scripts/startup: Permission denied`

The database exists with privileges on speedtest-tracker_test user.

**Comment by alexjustesen** [comment]
That is correct the current alpha release is no multi-arch. The code sitting in `main` has basic support for multi-arch but has not been tested yet. Looking to get that done over the weekend. 

#32

**Comment by benjamin-feron** [comment]
Allright !
thank you

**Comment by benjamin-feron** [comment]
Hi, do you have some news about the multi-arch compatibility ?

**Comment by alexjustesen** [comment]
I'm obviously behind on it, got some client work to wrap up before I circle back around to the issues backlog here. You might be able to pull in `main` and build the image locally (not from the releases) and give it a test.

My rpi's are offline right now, waiting on 1 more poe hat to come in before I put them back in the rack or I'd test it sooner.

**Comment by alexjustesen** [comment]
Closing as the initial issue was resolved in #51 and multi-arch will be addressed and is logged in another issue
